cli/tests/integration/lsp_tests.rs

```diff
@@ -6918,64 +6918,27 @@ fn lsp_completions_node_specifier() {
     .diagnostics
     .into_iter()
     .filter(|d| {
-      d.code
-        == Some(lsp::NumberOrString::String(
-          "import-node-prefix-missing".to_string(),
-        ))
+      d.code == Some(lsp::NumberOrString::String("no-cache-npm".to_string()))
     })
     .collect::<Vec<_>>();
 
-  // get the quick fixes
-  let res = client.write_request(
-    "textDocument/codeAction",
-    json!({
-      "textDocument": {
-        "uri": "file:///a/file.ts"
-      },
-      "range": {
-        "start": { "line": 0, "character": 16 },
-        "end": { "line": 0, "character": 18 },
-      },
-      "context": {
-        "diagnostics": json!(diagnostics),
-        "only": ["quickfix"]
-      }
-    }),
-  );
   assert_eq!(
-    res,
-    json!([{
-      "title": "Update specifier to node:fs",
-      "kind": "quickfix",
-      "diagnostics": [
-        {
-          "range": {
-            "start": { "line": 0, "character": 15 },
-            "end": { "line": 0, "character": 19 }
-          },
-          "severity": 1,
-          "code": "import-node-prefix-missing",
```

Comment:
I wonder how we can still have this quick fix. Perhaps it could be a warning displayed in the lsp for `"fs"` imports and then provide a quick fix.

Reply:
Now bare node specifiers produce lsp warnings with the quick fix

<img width="874" alt="Screenshot 2023-10-01 at 4 54 33" src="https://github.com/denoland/deno/assets/613956/96fce12e-4ada-495a-a932-aebfb24ec69e">

<img width="585" alt="Screenshot 2023-10-01 at 4 54 45" src="https://github.com/denoland/deno/assets/613956/73453476-ae63-4b65-b140-57b2642a7ec5">